sdk/storage/examples/blob_04.rs

```diff
@@ -44,8 +45,8 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     }
 
     let mut block_list = BlockList::default();
-    for id in block_ids.iter() {
-        block_list.blocks.push(BlobBlockType::Uncommitted(&id[..]));
+    for id in block_ids.into_iter() {
+        block_list.blocks.push(BlobBlockType::new_uncommitted(id));
```

Comment:
Why are we doing this? `block_list` is never used.

Reply:
Yes, it's weird, the example lacks the `PutBlockList` call. I've added it.